docs/admin/etcd_upgrade.md

```diff
@@ -5,210 +5,6 @@ assignees:
-   data.
-1. Based on the detected previous etcd version and the desired one
-   (communicated via environment variable), do the upgrade steps as
-   needed. This means that for every minor etcd release greater than the detected one and
-   less than or equal to the desired one:
-   1. Start etcd in that version.
-   1. Wait until it is healthy. Healthy means that you can write some data to it.
-   1. Stop this etcd. Note that this etcd will not listen on the default
-   etcd port. It is hard coded to listen on ports that the API server is not
-   configured to connect to, which means that API server won’t be able to connect
-   to it. Assuming no other client goes out of its way to try to
-   connect and write to this obscure port, no new data will be written during
-   this period.
-1. If the desired API version is v3 and the detected version is v2, do the offline
-   migration from the v2 to v3 data format. For that we use two tools:
-   * ./etcdctl migrate: This is the official tool for migration provided by the etcd team.
-   * A custom script that is attaching TTLs to events in the etcd. Note that etcdctl
-      migrate doesn’t support TTLs.
-1. After every successful step, update contents of the version file.
-   This will protect us from the situation where something crashes in the
-   meantime ,and the version file gets completely unsynchronized with the
-   real data. Note that it is safe if the script crashes after the step is
-   done and before the file is updated. This will only result in redoing one
-   step in the next try.
-
-All the previous steps are for the case where the detected version is less than or
-equal to the desired version. In the opposite case, that is for a rollback, the
-script works as follows:
-
-1. Verify that the detected version is 3.0.x with the v3 API, and the
-   desired version is 2.2.1 with the v2 API. We don’t support any other rollback.
-1. If so, we run the custom tool provided by etcd team to do the offline
-   rollback.  This tool reads the v3 formatted data and writes it back to disk
-   in v2 format.
-1. Finally update the contents of the version file.
-
-### Upgrade procedure
-Simply modify the command line in the etcd manifest to:
-
-1. Run the migration script. If the previously run version is already in the
-   desired version, this will be no-op.
-1. Start etcd in the desired version.
-
-Starting in Kubernetes version 1.6, this has been done in the manifests for new
-Google Compute Engine clusters. You should also specify these environment
-variables.  In particular,you must keep `STORAGE_MEDIA_TYPE` set to
-`application/json` if you wish to preserve the option to roll back.
-
-```
-TARGET_STORAGE=etcd3
-ETCD_IMAGE=3.0.17
-TARGET_VERSION=3.0.17
-STORAGE_MEDIA_TYPE=application/json
-```
-
-To roll back, use these:
-
-```
-TARGET_STORAGE=etcd2
-ETCD_IMAGE=3.0.17
-TARGET_VERSION=2.2.1
-STORAGE_MEDIA_TYPE=application/json
-```
+{% include user-guide-content-moved.md % }
```

Comment:
There's an extra space before the closing brace that is causing a syntax error and making Netlify fail:

{% include user-guide-content-moved.md %` `}